.buildkite/pipeline.yml

```diff
@@ -27,6 +27,12 @@ steps:
       - github_commit_status:
           context: Code formatting
 
+  - label: "🔍 Static code analysis"
+    command: ./gradlew detekt
+    notify:
+      - github_commit_status:
+          context: Static code analysis
+
```

Comment:
I meant to ask before: what's the advantage of using `Spotless` in this project? Does it do something extra from Detekt + ktlint as a formattter?